docs/docs/dev_docs/contracts/syntax/events.md

```diff
@@ -104,31 +126,11 @@ aztec-cli get-logs --from 5 --limit 1
-> A `ValueNote`, for example, currently uses 3 fields elements (plus the fixed overhead of 4). That's roughly `7 * 32 = 224` bytes of information, costing roughly 896 gas.
-
-> There are plans to compress encrypted note data further.
-> There are plans to adopt EIP-4844 blobs to reduce the cost of data submission further.
-
-## Processing events
-
-Both the encrypted and unencrypted events are stored in AztecNode.
-Unencrypted logs can be queried by anyone as we described above in the [Unencrypted Events](#unencrypted-events) section.
-
-Encrypted logs need to first be decrypted:
-
-### Decrypting
-
-One function of Aztec RPC Server is constantly loading encrypted logs from AztecNode and trying to decrypt them.
-When new encrypted logs are obtained, the Aztec RPC Server will try to decrypt them using the private encryption key of all the accounts registered inside Aztec RPC Server.
-If the decryption is successful, the Aztec RPC Server will store the decrypted note inside a database.
-If the decryption fails, the specific log will be discarded.
-
-For the Aztec RPC Server to successfully process the decrypted note we need to compute the note's 'note hash' and 'nullifier'.
-Aztec.nr enables smart contract developers to design custom notes, meaning developers can also customise how a note's note hash and nullifier should be computed. Because of this customisability, and because there will be a potentially-unlimited number of smart contracts deployed to Aztec, an Aztec RPC Server needs to be 'taught' how to compute the custom note hashes and nullifiers for a particular contract. Therefore, developers will need to implement a `compute_note_hash_and_nullifier` function inside their contracts.
-
-Every time a new note is successfully decrypted, the Aztec RPC Server will expect the existence of a `compute_note_hash_and_nullifier` function, which must teach it how to correctly process the new note.
+:::info
+the cost of submitting calldata to Ethereum is currently 4 gas per byte. Currently, in the Sandbox, an encrypted note has a fixed overhead of 4 field elements (to broadcast an ephemeral public key, a contract address, and a storage slot); plus a variable number of field elements depending on the type of note being emitted.
 
-This is an example implementation inside the `PrivateTokenContract`:
+A `ValueNote`, for example, currently uses 3 fields elements (plus the fixed overhead of 4). That's roughly `7 * 32 = 224` bytes of information.
 
-#include_code compute_note_hash_and_nullifier /yarn-project/noir-contracts/src/contracts/private_token_contract/src/main.nr rust
+- There are plans to compress encrypted note data further.
```

Comment:
Should this set of bullets be in a >

Reply:
It is inside an `:::info` so is looks like 
![image](https://github.com/AztecProtocol/aztec-packages/assets/16536249/e7d39098-dfea-4486-9daf-69d29fa57b7c)